mountpoint-s3-client/tests/common/mod.rs

```diff
@@ -44,6 +44,14 @@ pub fn get_test_bucket_without_permissions() -> String {
     std::env::var("S3_FORBIDDEN_BUCKET_NAME").expect("Set S3_FORBIDDEN_BUCKET_NAME to run integration tests")
 }
 
+pub fn get_secondary_test_region() -> String {
+    std::env::var("S3_SECONDARY_REGION").unwrap_or(String::from("ap-southeast-2"))
+ }
+ 
+pub fn get_test_domain() -> String {
+    std::env::var("S3_DOMAIN").unwrap_or(String::from("com"))
+ }
```

Comment:
Please use the full domain `amazonaws.com`.

Some of the US Government regions do not contain "amazonaws" at all.